build.sbt

```diff
@@ -9,6 +9,7 @@ import scala.xml.transform.{RewriteRule, RuleTransformer}
 // Global settings
 organization in ThisBuild := "org.http4s"
 version      in ThisBuild := scalazCrossBuild("0.15.10-SNAPSHOT", scalazVersion.value)
+relVersion   in ThisBuild := scalazCrossBuild("0.15.9", scalazVersion.value)
```

Comment:
I bet we could calculate this.

1. Use `VersionNumber` extractor (it's part of sbt) to break down `version`.
2. If last number is > 0, take last number - 1.
3. If last number is 0, just return the snapshot.

I guess that kind of sucks in the presence of milestone releases.

If we only published the tut on release builds, then we wouldn't need this flag.  And we'd be guaranteed that we weren't documenting unreleased features.  But then we can't improve the docs without a release.